tools/src/gt_mpi_gather.cc

```diff
@@ -385,19 +384,37 @@ void produce_column_histogram(const VariantQueryProcessor& qp, const VariantQuer
     histogram_op.equi_partition_and_print_bins(val);
 }
 
-int main(int argc, char *argv[]) {
-  //Initialize MPI environment
+void print_usage() {
+  std::cout << "Usage: gt_mpi_gather [options]\n"
+            << "where options include:\n"
+            << "\t \e[1m--help\e[0m, \e[1m-h\e[0m Print a usage message summarizing options available and exit\n"
+            << "\t \e[1m--json-config\e[0m=<query json file>, \e[1m-j\e[0m <query json file>\n"
+            << "\t\t Can specify workspace, array, query_column_ranges, query_row_ranges, vid_mapping_file,\n"
+            << "\t\t callset_mapping_file, query_attributes, query_filter, reference_genome, etc. as fields in the json file\n"
+            << "\t \e[1m--loader-json-config\e[0m=<loader json file>, \e[1m-l\e[0m <loader json file>\n"
+            << "\t\t Optional, if vid_mapping_file and callset_mapping_file fields are specified in the query json file\n"
+            << "\t \e[1m--workspace\e[0m=<workspace dir>, \e[1m-w\e[0m <GenomicsDB workspace dir>\n"
+            << "\t \e[1m--array\e[0m=<array dir>, \e[1m-A\e[0m <GenomicsDB array dir>\n"
+            << "\t \e[1m--print-calls\e[0m\n"
+            << "\t \e[1m--print-csv\e[0m\n"
+            << "\t \e[1m--print-Broad-GCVF\e[0m\n"
```

Comment:
should be --print-Broad-GVCF, not --produce-Broad-GCVF

Reply:
```
	 --help, -h Print a usage message summarizing options available and exit
	 --json-config=<query json file>, -j <query json file>
		 Can specify workspace, array, query_column_ranges, query_row_ranges, vid_mapping_file,
		 callset_mapping_file, query_attributes, query_filter, reference_genome, etc. as fields in the json file
	 --loader-json-config=<loader json file>, -l <loader json file>
		 Optional, if vid_mapping_file and callset_mapping_file fields are specified in the query json file
	 --workspace=<workspace dir>, -w <GenomicsDB workspace dir>
	 --array=<array dir>, -A <GenomicsDB array dir>
	 --print-calls
	 --print-csv
	 --produce-Broad-GCVF
		 --output-format=<output_format>, -O <output_format>
			 used with --print-Broad-GCVF
			 Output format can be one of the following strings: "z[0-9]" (compressed VCF),"b[0-9]" (compressed BCF)
			 or "bu" (uncompressed BCF). Default is uncompressed VCF if not specified.
	 --produce-histogram
	 --produce-interesting-positions
	 --version Print version and exit
```